src/com/company/EmployeeBook.java

```diff
@@ -0,0 +1,256 @@
+package com.company;
+
+public class EmployeeBook {
+
+    private Employee[] employee = new Employee[10];
+
+    public void employeeList(){
+        for (Employee employee : employee) {
+            if(employee != null ) {
+                System.out.println(employee.toString());
```

Comment:
можно упростить до
```suggestion
                System.out.println(employee);
```
Java умеет автоматом преобразовывать все объекты к типу String, неявно вызывая toString